### ISSUE TITLE
Metrics extension fails on input.tar.gz analysis

### ISSUE THREAD

**Issue by andygrunwald** [MetricsGrimoire/CVSAnalY#65]
If you extract the test/input.tar.gz and start a cvsanaly analysis on this with enabled Metrics extension, there is an error shown:

```
./cvsanaly2 --config-file ./config --db-database cvsanaly_before /Users/andygrunwald/Development/tmp/input
Parsing log for /Users/andygrunwald/Development/tmp/input (git)
Warning: Detected empty branch 'master', it'll be ignored
Executing extensions
Executing extension FileTypes
Executing extension Metrics
Error obtaining aaa/otherthing.renamed@51a3b654f252210572297f47597b31527c475fb8. Command ['git', 'show', u'51a3b654f252210572297f47597b31527c475fb8:aaa/otherthing.renamed'] returned 128 (fatal: Path 'aaa/otherthing.renamed' exists on disk, but not in '51a3b654f252210572297f47597b31527c475fb8'.
)
```

**Comment by linzhp** [comment]
This should have fixed in #60

**Comment by andygrunwald** [comment]
Fixed in #60
